peermusic/desktop
# Re add torrent

- [x] Only import one file at a time.
- [x] Start/stop torrent client on relevant sharing level changes.
- [x] <strike>Only add torrent to webtorrent client for downloading _once_.</strike> WebTorrent handles doublicates.
- [x] Remove download task from torrent client when download is cancelled.
- [x] Seed locally available files.

**Testing:**
1. Take a public domain music file: http://www.pdsounds.org/audio/download/780/Yah+rly.mp3
2. Associate two instances as devices.
3. Add the file to one of them.
4. Refresh the other.
5. Closer the one holding the file.
6. Set the remaining to sharing level "everyone".
7. Open another fresh instance (eg. incognito mode)
8. Add the the music file.
9. Set sharing level to "everyone".
10. Try to download in the other instance.

---

See: peermusic/user-stories/issues/58
Related: peermusic/peermusic/pull/36
